apps/bluetooth/js/modules/pair_manager.js

```diff
@@ -196,27 +151,31 @@ define(function(require) {
         return;
       }
 
-      this._defaultAdapter.pairingReqs.onpairingconsentreq = 
-        this._onPairingConsentReq.bind(this);
+      this._defaultAdapter.pairingReqs.addEventListener('pairingconsentreq',
+        this._constructPairingInfo.bind(this, 'consent'));
     },
 
     /**
-     * A handler to handle 'onpairingconsentreq' event while it's coming.
+     * A handler to construct pairing info while different pairing requests are
+     * coming.
      *
      * @access private
      * @memberOf PairManager
      */
-    _onPairingConsentReq: function(evt) {
-      Debug('onpairingconsentreq(): Pairing request from ' + 
-            evt.deviceName + ': pairing consent');
-      // TODO: Notify user of just-work pairing if user story is required.
-      throw new Error('Received pairing method "onpairingconsentreq". ' + 
-                      'It would need to implement if user story is required!!');
+    _constructPairingInfo: function(eventName, evt) {
```

Comment:
My original proposal is to handle one pairing event with one handler. Because I wonder platform might change the interface again. Then, we can keep the flexibility for each handler function. But it's good to modify them with your suggestion. It's more clear for me too. Thanks.